content/guides/use-case/pre-seeding.md

```diff
@@ -0,0 +1,335 @@
+
+   ```console
+   $ cat seed.sql | docker exec -i postgres psql -h localhost -U postgres -f-
+   ```
+
+   Once the query is executed, you will see the following results:
+
+   ```plaintext
+   CREATE DATABASE
+   You are now connected to database "sampledb" as user "postgres".
+   CREATE TABLE
+   INSERT 0 3
+   ```
+
+3. Run the following `psql` command to verify if the table named users is populated in the database `sampledb` or not. 
+
+   ```console
+   $ docker exec -it postgres psql -h localhost -U postgres sampledb
+   ```
+
+   You can now run `\l` in the `psql` shell to list all the databases on the Postgres server.
+
+   ```console
+   sampledb=# \l
+                                                List of databases
+   Name    |  Owner   | Encoding |  Collate   |   Ctype    | ICU Locale | Locale Provider |   Access privileges
+   -----------+----------+----------+------------+------------+------------+-----------------+-----------------------
+   postgres  | postgres | UTF8     | en_US.utf8 | en_US.utf8 |            | libc            |
+   sampledb  | postgres | UTF8     | en_US.utf8 | en_US.utf8 |            | libc            |
+   template0 | postgres | UTF8     | en_US.utf8 | en_US.utf8 |            | libc            | =c/postgres          +
+             |          |          |            |            |            |                 | postgres=CTc/postgres
+   template1 | postgres | UTF8     | en_US.utf8 | en_US.utf8 |            | libc            | =c/postgres          +
+             |          |          |            |            |            |                 | postgres=CTc/postgres
+   (4 rows)
+   ```
+
+   To retrieve all the data from the users table, enter the following query:
+
+   ```console
+   sampledb=# SELECT * from users;
+   id | name  |       email
+   ----+-------+-------------------
+    1 | Alpha | alpha@example.com
+    2 | Beta  | beta@example.com
+    3 | Gamma | gamma@example.com
+   (3 rows)
+   ```
+  
+   Use `\q` or `\quit` to exit from the Postgres interactive shell.
+
+## Pre-seeding the database by using Volumes to mount SQL files
+
+In Docker, mounting refers to making files or directories from the host system accessible within a container. This let you to share data or configuration files between the host and the container, enabling greater flexibility and persistence.
+
+Now that you have learned how to launch Postgres and pre-seed the database using an SQL script, it’s time to learn how to mount an SQL file directly into the Postgres containers’ initialisation directory (`/docker-entrypoint-initdb.d`). The `/docker-entrypoint-initdb.d` is a special directory in PostgreSQL Docker containers that is used for initializing the database when the container is first started
+
+### Stop the existing Postgres instance
+
+Make sure you stop any running Postgres containers (along with volumes) to prevent port conflicts before you follow the steps:
```

Comment:
````suggestion
Make sure you stop any running Postgres containers (along with volumes) to prevent port conflicts before you follow the steps:

```console
$ docker container stop postgres
```
````